Programs/python.c

```diff
@@ -15,6 +15,110 @@ wmain(int argc, wchar_t **argv)
 }
 #else
 
+/* Helpers to better handle the legacy C locale
```

Comment:
Even glibc devs consider 7-bit ASCII a legacy encoding at this point, it's just an enormous ecosystem to try to migrate to a new default way of doing things.

My view be different if we were planning to invest time and energy in getting Python 3 to "work properly" in the C locale, but the underlying assumptions are sufficiently incompatible that it isn't even clear what "working properly" would even mean. (PEP 540 gets closer than the status quo, but it's definition of "working properly" is "ignoring the declared locale encoding as almost certainly being wrong")